chip8.cpp

```diff
@@ -643,6 +637,10 @@ void chip8::decode(uint16_t _instruct){
                 std::string s = std::to_string(V[X]);
                 while(s.length()<3){s="0"+s;}
 
+                ram[I + 0] = s[0] - '0';
+                ram[I + 1] = s[1] - '0';
+                ram[I + 2] = s[2] - '0';
+
```

Comment:
This one was fixed on both branches, so the instruction is done two time, which is working but useless